### PR TITLE
clustergrammer

### PR DESCRIPTION
**Pull request by cristhomas**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/clustergrammer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - scikit-learn
+    - pandas
+
+test:
+  imports:
+    - clustergrammer
+
+about:
+  home: https://github.com/MaayanLab/clustergrammer-py
+  license: UNKNOWN
```
Comment (bgruening) [inline recipes/clustergrammer/meta.yaml]: https://github.com/MaayanLab/clustergrammer-py/blob/master/LICENSE

Its MIT licensed afaik.

File: recipes/clustergrammer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scikit-learn
+    - pandas
+
+  run:
+    - libgcc
```
Comment (bgruening) [inline recipes/clustergrammer/meta.yaml]: Is libgcc really needed here?

Comment (cristhomas) [reply recipes/clustergrammer/meta.yaml]: it was when I tested locally.

File: recipes/clustergrammer/meta.yaml (hunk 1 above)
Comment (cristhomas) [reply recipes/clustergrammer/meta.yaml]: thank you, missed that.

File: recipes/clustergrammer/meta.yaml (hunk 2 above)
Comment (cristhomas) [reply recipes/clustergrammer/meta.yaml]: as in, that's the only way it passed the mulled tests (and I tried a bunch of things)

Comment (bgruening) [reply recipes/clustergrammer/meta.yaml]: Yeah, this can be. If so then please add `- libgcc  # [linux]`

Comment (cristhomas) [reply recipes/clustergrammer/meta.yaml]: ok! thank you

**Pull request merged**